lib/core/test/unit/Cardano/Wallet/DB/SqliteSpec.hs

```diff
@@ -1244,6 +1255,17 @@ testMigrationPassphraseScheme = do
     Right walOldScheme     = fromText "4a6279cd71d5993a288b2c5879daa7c42cebb73d"
     Right walNoPassphrase  = fromText "ba74a7d2c1157ea7f32a93f255dac30e9ebca62b"
 
+testCreateMetadataTable ::
+    forall s k. (k ~ ShelleyKey, s ~ SeqState 'Mainnet k) => IO ()
+testCreateMetadataTable = do
```

Comment:
Nice

Reply:
I'll also test for the case when database version is newer than the one supported by the wallet library.